test/test_ops.py

```diff
@@ -140,7 +157,8 @@ def test_torch_fx_trace(self, device, x_dtype=torch.float, rois_dtype=torch.floa
     @pytest.mark.parametrize("seed", range(10))
     @pytest.mark.parametrize("device", cpu_and_gpu())
     @pytest.mark.parametrize("contiguous", (True, False))
-    def test_backward(self, seed, device, contiguous):
+    @pytest.mark.parametrize("deterministic", (False,))
+    def test_backward(self, seed, device, contiguous, deterministic):
```

Comment:
I think we can get rid of the parametrization here, and do as you did above in `test_forward()`, i.e. just add the param with a default

```suggestion
    def test_backward(self, seed, device, contiguous, deterministic=False):
```